contrib/better-defaults/keybindings.el

```diff
@@ -14,3 +14,5 @@
 
 ;; emacs state bindings
 (define-key evil-emacs-state-map (kbd "C-o") 'evil-execute-in-normal-state)
+(add-hook 'dired-mode-hook (lambda ()
+                             (local-set-key (kbd "M-o") 'dired-display-file)))
```

Comment:
Yeah. As @tuhdo said, `with-eval-after-load` works but a regular `eval-after-load` doesn't. This is what the `with-eval-after-load` expands to.